### ISSUE TITLE
Announcements for the 2.3 release

### ISSUE THREAD

**Issue by Holzhaus** [mixxxdj/website#77]
In addition to the usual beta and final release announcements, we came up with the idea to make a series of blogposts that highlight the major changes in Mixxx 2.3 and [discussed it on Zulip](https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/2.2E3.20planning.3A.20https.3A.2F.2Fgithub.2Ecom.2Fmixxxdj.2Fmixxx.2Fprojects.2F2/near/194929396).

Let's use this issue for planning these blogposts.

| Topic                                     | Author(s)             | PR           | Date
| ----------------------------------------- | --------------------- | ------------ | ------------
| 2.3 beta release annoncement              | @Be-ing               | #86          | 2020-06-07
| Auto DJ Improvements and Intro/Outro cues | @Be-ing               | #92          | 2020-07-09
| CMake                                     | @Holzhaus             | #80          | 2020-07-15
| ~~Overview Mouse interactions~~           | @Be-ing               |              | *(canceled)*
| Rekordbox Integration                     | @ehendrikd            | ~~#88~~ #125 | 2020-07-20
| Cue Colors                                | @Holzhaus / @Swiftb0y | #83          |  2020-08-25
| Track Colors                              | @Holzhaus             | #82          | 2020-10-24
| LateNight redesign                        | @ronso0               |              |
| Deck Cloning                              | @iamcodemaker         | #136         |
| Multithreaded Analysis                    | @uklotzde             |              |
| Sync Lock                                 | @ywwg                 |              | *(canceled)*
| Extended Metadata Support / MusicBrainz   | @uklotzde             |              |
| Serato Integration                        | @Holzhaus             | #81          |
| 2.3 final release annoncement             |                       |              |

**Note:** If you want to add/remove topics or want to change the author of a post, just comment below.

**Comment by ywwg** [comment]
Topic: Master Sync.  author: @ywwg

**Comment by Holzhaus** [comment]
> Topic: Master Sync. author: @ywwg

Added you to the list.

**Comment by Be-ing** [comment]
@ronso0, regarding the LateNight redesign, if you're up to it, it would be cool to do some archaeological work tracing the evolution of Mixxx's GUI with screenshots from past blog posts. On that note, I think we should include full resolution screenshots of every skin in each release blog post. We haven't done that in a long time, but it would be cool to be able to easily look back through Mixxx's history.

**Comment by Be-ing** [comment]
I propose that we end each of these blog posts with an encouragement for readers to get involved.

**Comment by iamcodemaker** [comment]
@Be-ing that's a good idea. One of us should write up a standard "Get involved!" blurb to put at the end of each post describing how this is an open source, community developed project and providing ways to participate. Maybe even including a list or link to a list of easy/starter issues. Here's a rough list of things that might make sense to include.
- fix bugs
- implement feature requests
- test new builds
- add mappings
- add skins

Might not make sense to include all of that stuff though...

**Comment by Holzhaus** [comment]
I reordered the posts in the table a bit, so that they can reference earlier posts and we don't get a bunch of subsequent posts by the same author.

**Comment by Holzhaus** [comment]
I'd like to have the first few post ready to publish before we release beta. I don't think that will be a big issue since the build servers are down anyway for the time being, but I think it makes sense to prioritize writing these posts over working on 2.4 for now.

**Comment by ehendrikd** [comment]
Just added #88 as a WIP for my Rekordbox post.

**Comment by Holzhaus** [comment]
@iamcodemaker @ronso0 As your posts are the first to go online, let please try to submit a draft PR for these posts soon-ish. If you don't have the time to do this, please let us know so that we can either reorder the posts or let someone else write it.

**Comment by Be-ing** [comment]
I didn't realize the table at the top of this PR implied any sort of order. Why does it matter what order they're posted?

**Comment by Holzhaus** [comment]
Because they might refer to previous posts. The Track Color PR refers to the Cue Color PR for example. And I think it probably makes sense to publish the Extended Metadata Support before the Serato metadata post because that feature builds upon that.

Also, LateNight makes sense to go first, because otherwise the screenshots in posts for other features would spoil the surprise and the actual LateNight post would probably become less interesting.

I also tried to make sure that we alternate between authors and nobody has 3 posts in a row.

**Comment by Be-ing** [comment]
I think we should just post them when they're ready.

**Comment by ronso0** [comment]
I'm about to write the LateNight Redesign blog post, and I think that's a good place to also mention some minor UI improvements that ease some library workflows enormously, like decks' track context menu, the double-click metadata popup (WIP), as well as the drag'n'drop feature. @iamcodemaker Or would you like to put that into the Deck Cloning post?

**Comment by Be-ing** [comment]
I think it would be better to discuss those in their own post.

**Comment by iamcodemaker** [comment]
@ronso0 I think it would be fine to talk about the drag and drop feature in relation to UI changes. But to @Be-ing's point, it doesn't make sense to go into much detail in the UI post.

**Comment by iamcodemaker** [comment]
I finished up the initial draft of the deck clone post: https://github.com/mixxxdj/website/pull/136.

**Comment by ronso0** [comment]
Okay, I finished some longstanding and urgent 2.3 PRs (as well as some personal backports ;), and now I'll have another look a my LateNight Redesign draft. Hope I can open a draft PR this week.

**Comment by Be-ing** [comment]
Ping @ronso0 do you have a draft for the LateNight blog post?

**Comment by ronso0** [comment]
nope, not yet, only a sketch with an outline I don't want to discuss. had to delay that repeatedly due to more important things.

I plan to get back to that next week.
